frame/composable-support/src/abstractions/block_fold.rs

```diff
@@ -95,12 +95,14 @@ where
 				let r = iterator.try_fold(
 					(0_u32, current_state),
 					|(processed, state), (key, value)| {
-						let next_state = f(state, key.clone(), value);
 						/* NOTE(hussein-aitlahcen):
```

Comment:
no sure, quite general comment, may not write  you name - it is quite clear and we do not need to call you at middle of night

Reply:
We do it for TODO's, which is more important since you need to know who made them when chasing down (git blame fails sometime with refactors etc).

For Note, I think we don't really need to, but I don't mind either